.github/workflows/publish_plugins.yml

```diff
@@ -0,0 +1,63 @@
+name: Publish Built-in Plugins
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  publish:
+    name: Publish plugins
+    runs-on: ubuntu-latest
+    env:
+      SCARB_REGISTRY_AUTH_TOKEN: ${{ secrets.SCARB_REGISTRY_AUTH_TOKEN }}
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: software-mansion/setup-scarb@v1
+        with:
+          scarb-version: ${{ github.ref_name }}
```

Comment:
Won't this be `refs/tags/<tag_name>`? https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#release

Reply:
Afaik, `ref_name` is a short name and is not the same as `GITHUB_REF`. 

`ref_name` should be a name of the tag that triggered the workflow: https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/accessing-contextual-information-about-workflow-runs#github-context